op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -67,33 +67,48 @@ func TestLoad(gt *testing.T) {
 	}()
 }
 
+func fundEOAs(num uint64, funder *dsl.Funder) []*dsl.EOA {
+	eoas := make([]*dsl.EOA, 0, num)
+	for range num {
+		eoas = append(eoas, funder.NewFundedEOA(eth.OneEther))
+	}
+	return eoas
+}
+
 func SpamInteropTxs(t devtest.T, numInitTxs uint64, source *L2, dest *L2, supervisor *dsl.Supervisor) {
 	var wg sync.WaitGroup
 	defer wg.Wait()
 	msgsCh := make(chan []types.Message, 100)
 	defer close(msgsCh)
 
+	// Mempool implementations may limit the number of concurrent transactions per account.
+	// We spam transactions from multiple EOAs to mitigate the possibility of mempool
+	// implementations being a limiting factor.
```

Comment:
Now we're at the other naive extreme where every tx is sent from a unique EOA. I think it should be ok, especially considering our goal to avoid overengineering, but I'm open to implementing a more complicated approach that reuses EOAs if it has advantages.

Reply:
Yeah, I suppose it could be a function of the gas limit. I'm trying to avoid funding EOAs after we start spamming since the funding txs could be delayed and they dilute the meaningful txs.